opm/simulators/linalg/PreconditionerFactory.hpp

```diff
@@ -111,13 +125,15 @@ class PreconditionerFactory
     // Helpers for creation of AMG preconditioner.
     static Criterion amgCriterion(const boost::property_tree::ptree& prm)
     {
-        Criterion criterion(15, prm.get<int>("coarsenTarget"));
+        Criterion criterion(15, prm.get<int>("coarsenTarget", 1200));
         criterion.setDefaultValuesIsotropic(2);
-        criterion.setAlpha(prm.get<double>("alpha"));
-        criterion.setBeta(prm.get<double>("beta"));
-        criterion.setMaxLevel(prm.get<int>("maxlevel"));
-        criterion.setSkipIsolated(false);
-        criterion.setDebugLevel(prm.get<int>("verbosity"));
+        criterion.setAlpha(prm.get<double>("alpha", 0.33));
+        criterion.setBeta(prm.get<double>("beta", 1e-5));
+        criterion.setMaxLevel(prm.get<int>("maxlevel", 15));
+        criterion.setSkipIsolated(prm.get<bool>("skip_isolated", false));
+	criterion.setNoPreSmoothSteps(prm.get<int>("pre_smooth", 1));
```

Comment:
Tabs should be replaced in this file.

Reply:
There you go. Hours spend for tab removal and then I introduce some more.... Will fix.